docs/physics/spectrum/basic.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>I think you can delete "(or with meter, nm, cm, etc. instead of Angstrom, depending on that unit you use for wavelength)". You don't make a similar statement for the other units (such as erg) and you also mention that "bins would now have the dimensions of length". I think it is clear enough without the additional statement (and easier to read).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>